packages/treasury/test/swingsetTests/governance/vat-zoe.js

```diff
@@ -0,0 +1,24 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+
+// noinspection ES6PreferShortImport
+import { E } from '@agoric/eventual-send';
+import { makeZoeKit } from '@agoric/zoe';
+
+export function buildRootObject(vatPowers) {
+  return Far('root', {
+    buildZoe: vatAdminSvc => {
+      const shutdownZoeVat = vatPowers.exitVatWithFailure;
+      const { zoeService, feeMintAccess } = makeZoeKit(
+        vatAdminSvc,
+        shutdownZoeVat,
+      );
+      const feePurse = E(zoeService).makeFeePurse();
+      const zoe = E(zoeService).bindDefaultFeePurse(feePurse);
+
+      console.log(`ZOE Vat  ${zoe}`);
```

Comment:
tests in this package seem pretty noisy, even without `-v`. I wonder if using `vatPowers.testLog` here would make it respect `-v`.

Reply:
The swingsetTests use the compare-to-golden approach. The logs that are relevant to the test result use `log`. `console.log()` should only be for debugging, which means it's my mistake that these are still here.  removed.